spec.html

```diff
@@ -70,7 +70,7 @@ <h1><ins>Runtime Semantics: HostImportModuleDynamically ( _referencingScriptOrMo
 
             <dd>
               <ul>
-                <li>At some future time, the host environment must perform FinishDynamicImport(_referencingScriptOrModule_, _specifier_, _promiseCapability_, NormalCompletion(*undefined*)).</li>
+                <li>At some future time, the host environment must perform FinishDynamicImport(_referencingScriptOrModule_, _specifier_, _promiseCapability_, Promise.resolve(*undefined*)).</li>
```

Comment:
this would probably need to be specified in terms of an abstract op (like `PromiseResolve`) rather than a global lookup

Reply:
Right, there's no intention of anyone doing such a dynamic lookup. This should not be taken literally.